package.json

```diff
@@ -92,17 +97,19 @@
     "strip-bom": "^2.0.0",
     "testcafe-browser-tools": "1.2.1",
     "testcafe-hammerhead": "10.6.7",
-    "testcafe-legacy-api": "2.0.2",
+    "testcafe-legacy-api": "3.0.0",
     "testcafe-reporter-json": "^2.1.0",
     "testcafe-reporter-list": "^2.1.0",
     "testcafe-reporter-minimal": "^2.1.0",
     "testcafe-reporter-spec": "^2.1.1",
     "testcafe-reporter-xunit": "^2.1.0",
     "time-limit-promise": "^1.0.2",
     "tree-kill": "^1.1.0",
+    "typescript": "^2.2.2",
```

Comment:
Is it possible to make typescript a peer dependency? Likely many projects are already going to have typescript pulled in as a dependency.